k8s/values.yaml

```diff
@@ -24,3 +24,10 @@ applications:
       - name: platform
         item: SHIBBOLETH_KEY
         path: "/etc/shibboleth/sp-key.pem"
+
+cronjobs:
+  - name: clear-expired-tokens
+    schedule: "0 0 * * 0"
```

Comment:
Could this be switched to 5 am? This time is set to UTC and 5 am UTC would get us to 1 am EDT, which is a bit safer in my opinion